core/src/test/scala/pureconfig/gen/package.scala

```diff
@@ -66,4 +67,14 @@ package object gen {
       dateTime <- ConfigurableSuite.localDateTimeArbitrary.arbitrary
     } yield new org.joda.time.DateTime(dateTime.getYear, dateTime.getMonthValue, dateTime.getDayOfMonth,
       dateTime.getHour, dateTime.getMinute, dateTime.getSecond, org.joda.time.DateTimeZone.UTC)
+
+  val genJavaBigDecimal: Gen[JavaBigDecimal] =
+    for {
```

Comment:
Consider changing this to the more compact and idiomatic `Arbitrary.arbitrary[BigDecimal].map(_.bigDecimal)`, it you prefer.